src/main/java/uk/gov/digital/ho/hocs/casework/domain/model/Stage.java

```diff
@@ -195,4 +158,12 @@ public boolean isActive() {
         return this.teamUUID != null;
     }
 
+    public void update(Map<String, String> newData, ObjectMapper objectMapper) {
```

Comment:
These methods are already part of `AbstractJsonDataMap` so why here?

Reply:
We can't extend 2 classes and we need to extend BaseStage as a @MappedSuperclass. I was about to suggest we move AbstractJsonDataMap to a static 'helper' class but I've just noticed it already is 'JsonMapUtils', 'AbstractJsonDataMap' is just a 1 line method to call the helper class so it doesn't need to exist in my opinion - we never pass Stage and CaseData into the same method that uses that base class for example. I'll remove it from both and call the helper class directly...quite why we need a helper class to use a map is for another day 😄